eclipsecs-sevntu-plugin/src/com/github/sevntu/checkstyle/checks/coding/checkstyle-metadata.properties

```diff
@@ -188,3 +188,10 @@ MoveVariableInsideIfCheck.desc = Checks if a variable is only used inside if sta
 
 RequireFailForTryCatchInJunitCheck.name = Require Fail For Try/Catch in JUnit Check
 RequireFailForTryCatchInJunitCheck.desc = Checks if a try/catch block has a junit fail assertion inside the try  for a junit method.
+
+Jsr305AnnotationsCheck.name=Jsr305AnnotationsCheck
+Jsr305AnnotationsCheck.desc=Checks method parameters and return values for the presence of @Nonnull, @Nullable, or @CheckForNull annotations.
+Jsr305AnnotationsCheck.packages=Packages to be checked.
+Jsr305AnnotationsCheck.excludePackages=Packages excluded from checking.
+Jsr305AnnotationsCheck.allowOverridingReturnValue=Allow Overriding return values, useful for upgrading.
+Jsr305AnnotationsCheck.allowOverridingParameter=Allow Overriding paramaters, useful for upgrading.
```

Comment:
Same here with more descriptive descriptions. What does allowing overrides actually do?

Reply:
I guess you had the file `sevntu-checkstyle-sonar-plugin/src/main/resources/com/github/sevntu/checkstyle/sonar/checkstyle-extensions.xml` in mind, right? Thank you, I have made the same text changes there, too.

Reply:
Yes, both files will have to change. One is for sonar and the other is for eclipse-cs.

Reply:
I don't see any updates to these 2 descriptions for `allowOverriding...`

Reply:
Yes, I did not change them. I thought it was clear, but I'll think of something tomorrow.

Reply:
Done with next commit.